packages/patternfly-4/react-core/src/components/BackgroundImage/BackgroundImage.js

```diff
@@ -7,31 +7,22 @@ import styles from '@patternfly/patternfly-next/components/BackgroundImage/backg
 /* eslint-disable camelcase */
 import {
   c_background_image_BackgroundImage_lg,
-  c_background_image_BackgroundImage_md,
-  c_background_image_BackgroundImage_md_2x,
   c_background_image_BackgroundImage_sm,
   c_background_image_BackgroundImage_sm_2x,
-  c_background_image_BackgroundImage_xl,
-  c_background_image_BackgroundImage_xs,
-  c_background_image_BackgroundImage_xs_2x,
   c_background_image_Filter
 } from '@patternfly/react-tokens';
 
 export const BackgroundImageSrc = {
   lg: 'lg',
   sm: 'sm',
   sm2x: 'sm2x',
-  xs: 'xs',
-  xs2x: 'xs2x',
   filter: 'filter'
 };
 
 const variableMap = {
   [BackgroundImageSrc.lg]: c_background_image_BackgroundImage_lg && c_background_image_BackgroundImage_lg.name,
   [BackgroundImageSrc.sm]: c_background_image_BackgroundImage_sm && c_background_image_BackgroundImage_sm.name,
   [BackgroundImageSrc.sm2x]: c_background_image_BackgroundImage_sm_2x && c_background_image_BackgroundImage_sm_2x.name,
-  [BackgroundImageSrc.xs]: c_background_image_BackgroundImage_xs && c_background_image_BackgroundImage_xs.name,
```

Comment:
Instead of xs and xs_2x looks like we need to replace with 
`c_background_image_BackgroundImage`
`c_background_image_BackgroundImage_2x`
And since these are smaller images, you probably want to keep the same xs and xs2x constants above